package.json

```diff
@@ -24,7 +26,7 @@
     "precommit": "lint-staged",
     "postcommit": "git update-index --again",
     "build": "gatsby build",
-    "develop": "gatsby develop",
+    "dev": "gatsby develop",
     "start": "yarn run develop",
```

Comment:
If you want to deviate from boilerplate - I think we'd have to change here too.

Reply:
nice catch, I'll change that